### DISCUSSION TITLE
Fehlermeldung nrf24l01 Modul wird nicht erkannt

### DISCUSSION THREAD

**Discussion by ThomasKlaus-ai** [tbnobody/OpenDTU#270] [General]
Guten Tag an die Runde,
ich bekomme meinen esp8266 open DTU nicht zum Laufen. Zwar sehe ich die Weboberfläche, erhalte aber keine Messwerte, weil es keine Verbindung zum NRF24L01 Modul gibt. Die Verdrahtung habe ich schon mehrfach geprüft, aber auch mein nrf24L01 + Modul bringt keine Verbesserung. Kann mir Jemand helfen ? Viele Grüße Thomas

**Comment by wib100** [comment]
Welche Meldungen kommen beim Starten auf der seriellen Konsole?

**Comment by ThomasKlaus-ai** [comment]
Mit welcher Software kann ich das erkennen ?Viele GrüßeThomas Am 25.10.2022 um 19:24 schrieb WiB ***@***.***>:﻿
Welche Meldungen kommen beim Starten auf der seriellen Konsole?

**Comment by madmartin** [comment]
Moin,
OpenDTU läuft nicht auf ESP8266 modulen.
Generell gilt (für ESP8266 und ESP32 module) für das "serial log": man nehme ein Terminalprogramm (z.B. "putty") konfiguriere es auf die serielle Schnittstelle des USB Konverters der auf dem ESP modul ist - Geschwindigkeit 115000 8 Bit no parity. Wenn nix zu sehen ist dann das ESP modul mal neu starten (reset knopf drücken).
Oder bei Verwendung von "vscode" gibts den Menüpunkt "Monitor".
Zum Funkmodul: es muß ein nrf24L01+ modul sein, das plus ist wichtig. Steht auch ganz deutlich in der Readme dieses Projekts. Mal mit der Lupe auf den Chip schauen.

**Comment by ThomasKlaus-ai** [comment]
Guten Morgen,vielen Dank für die Infos. In putty habe ich 115200, 8 bit, serial com3 eingestellt und den esp8266 neu gestartet und bekomme trotzdem keine Daten. Obwohl ich mit meinem Browser über die IP Adresse Kontakt habe.Viele GrüßeThomasAm 26.10.2022 um 09:55 schrieb Martin ***@***.***>:﻿
Moin,
OpenDTU läuft nicht auf ESP8266 modulen.
Generell gilt (für ESP8266 und ESP32 module) für das "serial log": man nehme ein Terminalprogramm (z.B. "putty") konfiguriere es auf die serielle Schnittstelle des USB Konverters der auf dem ESP modul ist - Geschwindigkeit 115000 8 Bit no parity. Wenn nix zu sehen ist dann das ESP modul mal neu starten (reset knopf drücken).
Oder bei Verwendung von "vscode" gibts den Menüpunkt "Monitor".
Zum Funkmodul: es muß ein nrf24L01+ modul sein, das plus ist wichtig. Steht auch ganz deutlich in der Readme dieses Projekts. Mal mit der Lupe auf den Chip schauen.

**Comment by ThomasKlaus-ai** [comment]
Hier sind die Zeilen aus Putty kopiert:
I: Devcontrol request 11 power limit 100
I: TX 19B Ch40 | 51 83 32 07 31 78 56 34 12 80 0B 00 03 E8 01 00 80 80 BF
I: enqueuedCmd: 5
Kann Jemand daran erkennen, warum es nicht funktioniert ?

**Reply by tbnobody** [reply]
Falsches Projekt. Die Ausgabe ist von Ahoy

**Comment by ThomasKlaus-ai** [comment]
Sorry für meinen falschen Kommentar.Viele GrüßeThomasAm 31.10.2022 um 13:49 schrieb tbnobody ***@***.***>:﻿
Falsches Projekt. Die Ausgabe ist von Ahoy

**Comment by ThomasKlaus-ai** [comment]
Hier habe ich weitere Meldungen. Es steht immer, daß der NRF nicht erreicht wird und ich die Verkabelung prüfen soll. Meine Verdrahtung stimmt aber mit den in GitHub beschriebenen Verkabelungen überein.
I: RF24 Amp Pwr: RF24_PA_I: MIN
I: Radio Config:
SPI Frequency           = 1 Mhz
                               Channel                  = 255 (~ 2655 MHz)
Model                   = nRF24L01+
RF Data Rate            = 1 MBPS
RF Power Amplifier      = PA_MAX
RF Low Noise Amplifier  = Enabled
CRC Length              = 16 bits
Address Length          = 5 bytes
Static Payload Length   = 32 bytes
Auto Retry Delay        = 4000 microseconds
Auto Retry Attempts     = 15 maximum
Packets lost on
                   current channel      = 15
Retry attempts made for
                           last transmission    = 15
Multicast               = Allowed
Custom ACK Payload      = Enabled
Dynamic Payloads        = Enabled
Auto Acknowledgment     = 0b111111
Primary Mode            = RX
TX address              = 0xffffffffff
pipe 0 ( open ) bound   = 0xffffffffff
pipe 1 ( open ) bound   = 0xffffffffff
pipe 2 ( open ) bound   = 0xff
pipe 3 ( open ) bound   = 0xff
pipe 4 ( open ) bound   = 0xff
pipe 5 ( open ) bound   = 0xff
W: WARNING! your NRF24 module can't be reached, check the wiring
I: [NTP]: 2022-11-05 11:48:29
I: subscribe to HM400 Garten/devcontrol/#
I: Requesting Inverter SN 112183320731
I: Requesting Inverter SN 112183320731
I: Requesting Inverter SN 112183320731
I: Power limit for inverter 0 set to 100W
I: Power Limit Control Setting 256
I: Requesting Inverter SN 112183320731
I: Devcontrol request 11 power limit 100
I: TX 19B Ch3 | 51 83 32 07 31 78 56 34 12 80 0B 00 03 E8 01 00 80 80 BF
I: enqueuedCmd: 5
I: Requesting Inverter SN 112183320731
I: Devcontrol request 11 power limit 100
I: TX 19B Ch23 | 51 83 32 07 31 78 56 34 12 80 0B 00 03 E8 01 00 80 80 BF
I: enqueuedCmd: 5
I: Power limit for inverter 0 set to 100W
I: Power Limit Control Setting 256
I: Requesting Inverter SN 112183320731
I: Devcontrol request 11 power limit 100
I: TX 19B Ch40 | 51 83 32 07 31 78 56 34 12 80 0B 00 03 E8 01 00 80 80 BF
I: enqueuedCmd: 5
I: Requesting Inverter SN 112183320731
I: Devcontrol request 11 power limit 100
I: TX 19B Ch61 | 51 83 32 07 31 78 56 34 12 80 0B 00 03 E8 01 00 80 80 BF
I: enqueuedCmd: 5

**Reply by tbnobody** [reply]
Diese Meldungen stammen von ahoy. Nicht von opendtu. Falsches Projekt 😀

**Reply by stefan123t** [reply]
Interessant er meldet RF data rate 1MBps obwohl er eigentlich auf 250kbps senden & empfangen soll. Bitte unter https://www.github.com/lumapu/ahoy/issues ein Issue aufmachen und unser Issue Formular mit Bildern und Pin Belegung etc. ausfüllen. Danke!

**Comment by JohnUps** [comment]
Hallo

Habe das gleiche Verhalten.
Verwendet: ESP32-WROOM
Betitelung bei AliExpress 
ESP32 TYPE-C USB CH340C CP2102 WiFi + Bluetooth Ultra-Low Power Dual Core ESP32-DevKitC-32 ESP-WROOM-32 Expansion Board

Software: AHOY
You are using the current stable release: 0.7.36

Meldung:
your NRF24 module can't be reached, check the wiring, pinout and enable

Habe 2 NRF24+ Module aus der gleichen Sendung von AliExpress: Gleiches Problem bei beiden Modulen.
(das kleine + ist mit der Lupe auch erkennbar)
NRF24L01+ 2.4G wireless data transmission module 2.4GHz NRF24L01 upgrade version NRF24L01+PA+LNA 1000 Meters For Arduino

Verkabelung mehrfach kontrolliert - Ebenfalls wie oben: Umstellung des GPIO 17 auf 4 erst später bemerkt, dann in AHOY Konfiguration angepasst. Trotzdem meldet AHOY (nach Neustart) keine Verbindung.

Irgendwo war zu lesen, dass ggf. gefakete NRF24L01 als "+" Varianten gelabelt aufgetreten sind.
Kein Kondensator verbaut - Daher das (ebenfalls nicht funktionierende Display abgeklemmt. 

Keine Änderung ...
Hat jemand Ideen?

--------------------------------------------------
System Information
mac
B0:A7:32:DC:.....
sdk
v4.4.4
cpu_freq
240
heap_free
172852
sketch_used
1154
wifi_rssi
-57
ts_uptime
132
version
0.7.36
esp_type
ESP32
heap_total
290996
chip_revision
3
chip_model
ESP32-D0WDQ5
chip_cores
2
Radio
nrf24l01+
is not connected
TX count
0
RX success
0
RX fail
0
RX no answer
0
RX fragments
0
TX retransmits
0
Sun
(entfernt)
Night Communication
enabled
